src/proto/parameters.proto

```diff
@@ -293,6 +294,29 @@ message RobotNavigationObstacleConfig
     ];
 }
 
+message HRVOConfig
+{
+    required double linear_velocity_kp = 1 [
+        default                   = 2.0,
+        (bounds).min_double_value = 0.001,
+        (bounds).max_double_value = 10.0
+    ];
+
+    // The maximum amount which the PID velocity can be greater than
+    // the actual robot velocity. Used to prevent swinging [m/s]
+    required double linear_velocity_max_pid_offset = 2
+        [default = 0.3, (bounds).min_double_value = 0.0, (bounds).max_double_value = 5.0];
+
```

Comment:
The main goal of this was to stop the proportional controller from spewing out a super large velocity which causes the robots from swinging when the destination changes. 
The problem I noticed with just having the PID velocity be current velocity + max accel. was that the robot would go around obstacles much slower than before. I think this happens because we're projecting a smaller preferred velocity on the side of the velocity obstacles which makes the new velocity even smaller. So I decided to use a velocity slightly larger than max acceleration over 1 frame as how much I will allow the PID to offset, so it minimizes swinging while also allows the robot to go around obstacles slightly faster. Though to be fair, the current value of max 0.3m/s offset was mostly a guess, and it's not necessarilly an optimal value.

Reply:
this is fine for now but imo in the future, we should be adjusting preferred velocity calculation to pick points around obstacles that are less tight and faster